app/src/main/java/org/qosp/notes/ui/reminders/ReminderManager.kt

```diff
@@ -92,6 +95,10 @@ class ReminderManager(
         reminderRepository.getById(reminderId).first()?.let { notificationTitle = it.name }
         reminderRepository.deleteById(reminderId)
 
+        if (notificationTitle.isEmpty()) {
+            noteRepository.getById(noteId).first()?.let { notificationTitle = it.title }
```

Comment:
I forgot to ask. Is there a firstOrNull() option here? Does it error out when there is no record?

Reply:
I'm new to Android development. If I missed anything, please correct it.
I thought a reminder is created attaching to a note, so there should be a note matching the `noteId` of the reminder.
I copied the expression from:
https://github.com/quillpad/quillpad/blob/52713ecb402a4c14e74539f60327882f5fc14884/app/src/main/java/org/qosp/notes/ui/attachments/dialog/AttachmentDialogViewModel.kt#L25
https://github.com/quillpad/quillpad/blob/52713ecb402a4c14e74539f60327882f5fc14884/app/src/main/java/org/qosp/notes/data/repo/TagRepository.kt#L74